doc/releases.md

```diff
@@ -54,25 +54,28 @@ Run a **[iojs+any-pr+multi](https://jenkins-iojs.nodesource.com/job/iojs+any-pr+
 
 ### 2. Produce a Nightly Build _(optional)_
 
-If there is reason to produce a test release for the purpose of having others try out installers or specifics of builds, produce a nightly build using **[iojs+release+nightly](https://jenkins-iojs.nodesource.com/job/iojs+release+nightly/)** and wait for it to drop in <http://iojs.org/download/nightly/>.
+If there is reason to produce a test release for the purpose of having others try out installers or specifics of builds, produce a nightly build using **[iojs+release](https://jenkins-iojs.nodesource.com/job/iojs+release/)** and wait for it to drop in <http://iojs.org/download/nightly/>. Follow the directions and enter a proper length commit sha, a date string and and select "nightly" for "disttype".
 
 This is particularly recommended if there has been recent work relating to the OS X or Windows installers as they are not tested in any way by CI.
 
-### 4. Update the _CHANGELOG.md_
+### 3. Update the _CHANGELOG.md_
 
-Use the following git command to produce a list of commits since the last release:
+Collect a formatted list of commits since the last release. Use [changelog-maker](https://github.com/rvagg/changelog-maker) to do this.
 
 ```
-git log --pretty=format:"* [%h] - %s (%aN)" \
-  --since="$(git show -s --format=%ad `git rev-list --tags --max-count=1`)"
+$ changelog-maker --group
 ```
 
-_(You will probably need to omit the first two commits as they relate to the last release.)_
+Note that changelog-maker counts commits since the last tag and if the last tag in the repository was not on the current branch you may have to supply a `--start-ref` argument:
+
+```
+$ changelog-maker --group --start-ref v2.3.1
+```
```

Comment:
there's been two major revisions to how it works since early on and I find it quite reliable now. I used it for 2.3.2 because the new 3.0.0-rc1 patch got in the way and it worked perfectly using the exact command above, it just didn't trim off the release commit and "working on" commit but that's by design with `--start-ref`